pyfstat/helper_functions.py

```diff
@@ -41,48 +43,80 @@ def set_up_matplotlib_defaults():
     plt.rcParams["axes.formatter.useoffset"] = False
 
 
-def set_up_command_line_arguments():
-    """Parse global commandline arguments."""
-    parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "-v",
-        "--verbose",
-        action="store_true",
-        help="Increase output verbosity [logging.DEBUG]",
-    )
-    parser.add_argument(
-        "-q",
-        "--quite",
-        action="store_true",
-        help="Decrease output verbosity [logging.WARNING]",
-    )
-    args, unknown = parser.parse_known_args()
+def set_up_logger(outdir=None, label="pyfstat", log_level="INFO"):
+    """Setup the logger.
 
-    logger = logging.getLogger()
-    stream_handler = logging.StreamHandler()
-    stream_handler.setFormatter(
-        logging.Formatter("%(asctime)s %(levelname)-8s: %(message)s", datefmt="%H:%M")
-    )
+    Based on the implementation in Nessai:
+    https://github.com/mj-will/nessai/blob/main/nessai/utils/logging.py
+
+    Parameters
+    ----------
+    outdir : str, optional
+        Path to outdir directory.
+    label : str, optional
+        Label for this instance of the logger.
```

Comment:
I don't think that's true. It seems to only get used for the file name. While the logger instance is always called "pyfstat". And actually I'd say that's fine, as the logger instance name gets written on every log line, which would get spammy with run labels.